src/main/config/docker-compose.yaml.ctmpl

```diff
@@ -0,0 +1,22 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{$keyname := printf "secret/dsde/%s/firecloud-ui/docker-compose.yaml" $environment}}
+{{with vault $keyname}}
+
+ui:
+  image: {{.Data.ui_image}}
+  ports:
+    - "80:80"
+    - "443:443"
+    {{.Data.ui_additional_ports}}
+  {{.Data.ui_dns}}
+  {{.Data.ui_volumes}}
+  environment:
+    GOOGLE_CLIENT_ID: {{.Data.env_client_id}}
+    ORCH_URL_ROOT: {{.Data.env_orchestration_url}}
```

Comment:
As far as I can tell, this is the only harm done to our use cases. Now, developing against a local instance of orchestration is not as simple as setting this environment variable before calling `start-server.sh`. What is the path forward for this use case?